docs/integration/guides/text-to-speech.md

```diff
@@ -33,12 +33,12 @@ exten => 123,n,agi(googletts.agi,"${TEXT}",en) ; Execute the googletts agi scrip
 ```yaml title="Service"
 service: asterisk.originate
 data:
-    channel: PJSIP/100            # The device it will call
+    channel: PJSIP/100            # The device it will call (either PJSIP or SIP --> depends how your device [extension] has been created) PJSIP for pjsip                                     extension / SIP for chan_sip extensions
```

Comment:
There are much more type of devices in Asterisk. Don't think it's a good idea to list that here.